src/interfaces/wallet.cpp

```diff
@@ -463,6 +463,7 @@ class WalletImpl : public Wallet
     }
     unsigned int getConfirmTarget() override { return m_wallet.m_confirm_target; }
     bool hdEnabled() override { return m_wallet.IsHDEnabled(); }
```

Comment:
In commit "Set a wallet flag for blank wallets" (4d2b8b4d93cb979fba81182258d93d562608a632)

I don't get understand the GUI part of this change at all. Does it actually make sense to show an HD enabled logo for a wallet that doesn't have an HD key? And will generate non-HD keys if an address is requested (IIUC)?